library/deleteDrug.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ *  @package   OpenEMR
+ *  @link      http://www.open-emr.org
+ *  @author    Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @copyright Copyright (c )2020. Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ *
+ *
+ */
+
+require_once "../interface/globals.php";
+
+use OpenEMR\Common\Csrf\CsrfUtils;
+
```

Comment:
this script should get an acl check on it

Reply:
This is why I don't like to review code! I never remember this stuff!:)
Then again: umm what did I have for breakfast this AM?

Reply:
the reason that I did not do an acl here is because the delete function can only be access from the prescription page which is acl checked. Only providers can see the delete function. Why would there need to be an acl on this script?